app/js/patchShow.js

```diff
@@ -85,6 +85,9 @@ function getUrl() {
 //  4. Write the patch to patch.json
 //  5. Apply the patch to the entire show
 function patchShow() {
+    if (!navigator.onLine) {
```

Comment:
I would move this into the next block of code directly under the
`label.innerText = 'Fetching Google Sheet...';` line because this is part of that whole process.  That way, errors thrown here should be caught by the promise.

I would also include a `return;` at the end of this `if` statement because we don't want to try and execute a request over the Internet when we know there's no Internet go send a request over

Reply:
You'll probably need to delete your local branch and checkout to a branch of the same name because @ByronAmbright force pushed to this branch to resolve some lingering commit issues.  Let us know if you need help figuring out how to do that.